deployment/cloud-run/test_routing_fixed.py

```
@@ -14,44 +15,72 @@
 
 try:
     from secure_api_server import app
-    print("✅ Successfully imported secure_api_server")
+    print("Successfully imported secure_api_server")
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)